### PR TITLE
grid

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
- review please
- [code sandbox Link](https://e49n2b.csb.app/grid.html)

### CODE REVIEW

File: grid.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.side-bar{
+    grid-column: 1/2;
+    grid-row: 2/5;
+}
+.side-bar a{
+    text-decoration: none;
+    color: black;
+    font-size: 22px;
+}
+.side-bar div{
```
Comment (poojakelkar) [inline grid.css:48]: Add some margin-left to sidebar

File: grid.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
+.container{
+    display: flex;
+    background-color: lightblue;
+    padding: 15px;
```
Comment (poojakelkar) [inline grid.css:4]: Use rem instead of px

**Review by poojakelkar — COMMENTED** [review]
added some changes, otherwise looks good to merge.

File: grid.css (hunk 1 above)
Comment (RaghavendraaReddy) [reply grid.css:48]: will do that

File: grid.css (hunk 2 above)
Comment (RaghavendraaReddy) [reply grid.css:4]: okay

**Review by Rajitha04 — APPROVED** [review]
Try to avoid using px for larger values , rest looks fine

**Pull request merged**